src/app/showcase/doc/treeselect/basicdoc.ts

```diff
@@ -10,7 +10,7 @@ import { NodeService } from '../../service/nodeservice';
             <p>In single selection mode, value binding should be the <i>key</i> value of a node.</p>
         </app-docsectiontext>
         <div class="card flex justify-content-center">
-            <p-treeSelect class="md:w-20rem w-full" containerStyleClass="w-full" [(ngModel)]="selectedNodes" [options]="nodes" placeholder="Select Item"></p-treeSelect>
+            <p-treeSelect class="md:w-20rem w-full" [showClear]="true" containerStyleClass="w-full" [(ngModel)]="selectedNodes" [options]="nodes" placeholder="Select Item"></p-treeSelect>
```

Comment:
why you added the property `[showClear]` in `treeSelect`. I think is not necessary add that